symplyphysics/laws/electricity/maxwell_equations/gauss_law.py

```diff
@@ -0,0 +1,56 @@
+from sympy import Eq, solve, sqrt, log, pi, sin, Expr, Derivative
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output, dimensionless, Function, Vector, SI)
+from symplyphysics.core.fields.operators import divergence_operator as div
+from symplyphysics.core.coordinate_systems.coordinate_systems import CoordinateSystem
+from symplyphysics.core.fields.vector_field import VectorField
+from sympy.physics.units import electric_constant
+
+## Description
+## The divergence of electrical induction at some point is equal to the bulk charge density at that
+## point. A simpler formulation of the law is that electric charges exist.
+
+## Law is: div(D(x, y, z)) = p, where
+## D - the vector of electrical induction,
+## x, y, z - coordinates in a rectangular coordinate system,
+## p - the bulk density of the electric charge,
+## div - divergence (the sum of partial derivatives in coordinates).
+
+electrical_induction_x = Function("D_x", units.charge / units.area)
+electrical_induction_y = Function("D_y", units.charge / units.area)
+electrical_induction_z = Function("D_z", units.charge / units.area)
+x = Symbol('x', units.length)
+y = Symbol('y', units.length)
+z = Symbol('z', units.length)
+bulk_density_of_electric_charge = Symbol('bulk_density_of_electric_charge', units.charge / units.volume)
+
+law = Eq(Derivative(electrical_induction_x(x, y, z), (x, 1)) + Derivative(electrical_induction_y(x, y, z), (y, 1)) + Derivative(electrical_induction_z(x, y, z), (z, 1)), bulk_density_of_electric_charge)
```

Comment:
We have divergence operator for this case:
https://github.com/blackyblack/symplyphysics/blob/dbdb652311e1c7525c1ac44771e5972c0eebc4d0/symplyphysics/core/fields/operators.py#L45
It is applicable to a vector field, not a single vector.